tests/unit/get-value-test.js

```diff
@@ -57,7 +57,6 @@ module('Unit | get value', function() {
       let value = getValue({ context, macro, key });
 
       assert.strictEqual(callback.thisValues[0], context);
-      assert.deepEqual(callback.args[0], [key]);
```

Comment:
Was this an accidental removal?

Reply:
No. The code previously used `@ember/object`'s `computed`, which passes the property key as the first argument to the getter. But the code change in this PR makes it so that `computed` is never passed to `getValue`, but `ember-macro-helpers`' `computed` is instead, and `ember-macro-helpers`' `computed` does not pass the property key to the getter.